src/xtopcom/xdb/xdb_face.h

```diff
@@ -25,7 +25,7 @@ class xdb_transaction_t {
 
 class xdb_face_t {
  public:
-    virtual bool open() = 0;
+    virtual bool open(bool readOnly = false) = 0;
```

Comment:
must add default value 'false'?
cannot be `virtual bool open(bool readOnly) = 0`?

for virtual member function, it's recommend not provide default value.

Reply:
![image](https://user-images.githubusercontent.com/2065762/123414484-85dd4f80-d5e6-11eb-80f6-3ce58fd6ce26.png)